.github/workflows/auto-updater.yml

```diff
@@ -0,0 +1,46 @@
+name: Automatic Data Update
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  DataUpdater:
+    name: Automatic Data Update
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+      - name: Use Node.js v18
+        uses: actions/setup-node@17f8bd926464a1afa4c6a11669539e9c1ba77048 # tag=v3
+        with:
+          node-version: 18
+          cache: yarn
+          registry-url: https://registry.yarnpkg.com/
+      - name: Install Dependencies
+        run: yarn --immutable
+      - name: Build packages for script
+        run: yarn turbo run build --filter=@sapphire/time-utilities --filter=@sapphire/fetch
+      - name: Run updater
+        run: node --experimental-json-modules scripts/twemoji-regex-updater.mjs
+      - name: Run prettier on the code
+        run: yarn format
+      - name: Commit any changes and create a pull request
+        env:
+          GITHUB_USER: github-actions[bot]
+          GITHUB_EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git add .;
+          if git diff-index --quiet HEAD --; then
+            echo "No changes to commit, exiting with code 0"
+            exit 0;
+          else
+            git remote set-url origin "https://${GITHUB_TOKEN}:x-oauth-basic@github.com/${GITHUB_REPOSITORY}.git";
```

Comment:
does this work?

Reply:
Yes I use it with graphql-pokemon https://github.com/favware/graphql-pokemon/runs/6766851881?check_suite_focus=true

Reply:
Your hooks there aren't executable btw 😄